config/config.js

```diff
@@ -4,7 +4,7 @@ const config = {
         DATABASE: process.env.MONGODB_URI
     },
     default: {
-        SECRET: 'bt9#174^vyp4kugv$j$ah1cg6b2dt5p1@1a*p=gpv36gr2hbuw',
+        SECRET: process.env.SECRET_KEY,
```

Comment:
consider using the `dotenv` package and a .env file. So, this config file pretty much just creates an object from the values in `process.env.*`